dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityInstanceController.java

```diff
@@ -287,6 +271,66 @@ public void getAttributeImage(
         }
     }
 
+    @GetMapping( value = "{teiId}/{attributeId}/files" )
```

Comment:
If in the response there will always be one file, maybe we should change the URL
```suggestion
    @GetMapping( value = "{teiId}/{attributeId}/file" )
```